### DISCUSSION TITLE
Where are normalized counts in Signac?

### DISCUSSION THREAD

**Discussion by Ryuichi30** [stuart-lab/signac#1472] [Q&A] [answered]
Please let me know where normalized counts (that is TF-IDF normalized counts) are in Signac object.
I would like to plot a heatmap using some of the normalized counts.

Thank you,
Ryuichi

**Comment by timoast** [comment, marked as answer]
Normalized data is stored in the data slot, you can access it using `GetAssayData(object, slot = "data")`. Please see the Seurat documentation for more information: https://github.com/satijalab/seurat/wiki/Assay#slots

**Reply by Ryuichi30** [reply]
Thank you very much for your information.

Best,
Ryuichi
